modules/apps/frontend-js/frontend-js-spa-web/src/main/resources/META-INF/resources/init.js

```diff
@@ -17,6 +19,31 @@ import {getUrlPath} from './util/utils';
 const initSPA = function (config) {
 	const app = new App(config);
 
+	const checkExcludedTargetPortlets = (uri) => {
+		const id = uri.searchParams.get('p_p_id');
+
+		if (!id || !config.excludedTargetPortlets) {
+			return true;
+		}
+
+		if (config.excludedTargetPortlets.includes(id)) {
+			return false;
+		}
+		else {
+			const redirect = uri.searchParams.get(
+				getPortletNamespace(id) + 'redirect'
+			);
```

Comment:
What about other params that can contain URL, like `backURL`? Do we maybe want to loop over all params, and check each one that regex-matches an URL?

Reply:
see https://github.com/liferay-frontend/liferay-portal/pull/4936